### PR TITLE
Update changelog for version v0.15.2

### PR DESCRIPTION
**Pull request by cxbrowne1207**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/changelog.md [hunk 1]
```diff
@@ -13,6 +25,9 @@ weight: 35
 | Bottlerocket | 1.13.1  | 1.13.1    |  Not supported     | Not supported	     | Not supported   |
 | RHEL         | 8.7	   | 8.7	     |  Not supported     | 8.7	               | Not supported   |
 
+### Added
+- Kubernetes 1.26 support
+
 ### Upgraded
 - Cilium updated from version `v1.11.10` to version `v1.11.15`
```
Comment (panktishah26) [inline docs/content/en/docs/reference/changelog.md:32]: Can we add this line in the changelog?

> ### Fixed
> Fix tinkerbell workload cluster reconciliation issue

File: docs/content/en/docs/reference/changelog.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ linkTitle: "What's New?"
...
+|    :---:     |  :---:  |   :---:   |   :---:  |    :---:   | :---: |
+| Ubuntu       | 20.04	 | 20.04     |	20.04             | Not supported	     | 20.04 |  
+| Bottlerocket | 1.13.1  | 1.13.1    |  Not supported     | Not supported	     | Not supported   |
+| RHEL         | 8.7	   | 8.7	     |  Not supported     | 8.7	               | Not supported   |
+
+### Added
+- Support for no-timeouts to more EKS Anywhere operations (#[5565](https://github.com/aws/eks-anywhere/issues/5565))
+
+### Fixed
+- Fix deleting Tinkerbell workload cluster due to incorrect SSH key update during reconciliation (#[5554](https://github.com/aws/eks-anywhere/issues/5554))
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/changelog.md:20]:
```suggestion
- Fixed the failure to delete a Tinkerbell workload cluster due to an incorrect SSH key update during reconciliation (#[5554](https://github.com/aws/eks-anywhere/issues/5554))
```

**Review by chrisnegus — COMMENTED** [review]
Suggested a small working change.

**Review by taneyland — APPROVED** [review]

**Pull request merged**